browser/about_flags.cc

```diff
@@ -428,7 +428,14 @@ constexpr char kRestrictWebSocketsPoolDescription[] =
 #define BRAVE_SHIELDS_V2_FEATURE_ENTRIES
 #endif
 
+#if defined(OS_ANDROID)
+#define BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES_TRAILING_COMMA ,
+#else
+#define BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES_TRAILING_COMMA
+#endif  // defined(OS_ANDROID)
+
 #define BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES                                   \
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES_TRAILING_COMMA                        \
```

Comment:
let's move `BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES` location in the upstream file instead, it can be put as a first item. We don't need to add `BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES_TRAILING_COMMA`.

Reply:
Moved in 3d17dcdb5cd649adc08ae6b99815aff2e68356ad